backend/src/models/User.ts

```diff
@@ -51,8 +51,8 @@ class User extends Model<User> {
     },
   })
   @Unique
-  @Column
-  phoneNumber: string;
+  @Column(DataType.TEXT)
```

Comment:
required for type inference for sequelize to create db as it is ambiguous via typescript

Reply:
Interesting, what do you see? I was certain that enabling `    "emitDecoratorMetadata": true,` in tsconfig was enough for this.

Reply:
Discussed in slack, this is needed because of the new `| null` type, which is needed because we actually assign this property to `null`.